eth-types/src/evm_types.rs

```diff
@@ -75,6 +75,10 @@ impl GasCost {
     pub const CALL_WITH_VALUE: u64 = 9000;
     /// Constant cost for turning empty account into non-empty account
     pub const NEW_ACCOUNT: u64 = 25000;
+    /// Gas cost of warming up an account with the access list
+    pub const ACCESS_LIST_ADDRESS: u64 = 2400;
+    /// Gas cost of warming up a storage with the access list
+    pub const ACCESS_LIST_STORAGE: u64 = 1900;
```

Comment:
Can't find in this PR for where uses this 2 variables

Reply:
Good catch. I will remove them.